src/metadata/EntityMetadata.ts

```diff
@@ -485,14 +486,18 @@ export class EntityMetadata {
     /**
      * Creates a new entity.
      */
-    create(): any {
+    create(queryRunner: QueryRunner = this.connection.createQueryRunner()): any {
```

Comment:
no! it should be optional means you should be `queryRunner?: QueryRunner`. Creating a query runner is a responsible task, don't do that until you absolutely need it.